docs/getting-started.rst

```diff
@@ -58,100 +58,97 @@ long format, i.e. each person no matter what age has his/her own row.
++---------------------+--------------------------------------------------------+-------+
+| rentner             | Dummy: Pensioner employment status                     | bool  |
++---------------------+--------------------------------------------------------+-------+
+| betreuungskost_m    | Monthly childcare expenses                             | float |
++---------------------+--------------------------------------------------------+-------+
+| miete_unterstellt   | Monthly value of owner-occupied housing                | float |
++---------------------+--------------------------------------------------------+-------+
+| kapital_eink_m      | Monthly capital income                                 | float |
++---------------------+--------------------------------------------------------+-------+
+| vermiet_eink_m      | Monthly rental income                                  | float |
++---------------------+--------------------------------------------------------+-------+
+| kaltmiete_m         | Monthly rent expenses (without heating)                | float |
++---------------------+--------------------------------------------------------+-------+
+| heizkost_m          | Monthly heating expenses                               | float |
++---------------------+--------------------------------------------------------+-------+
+| jahr_renteneintr    | Statutory retirement age (might be in the future)      | int   |
++---------------------+--------------------------------------------------------+-------+
+| behinderungsgrad    | Handicap degree (between 0 and 100)                    | int   |
++---------------------+--------------------------------------------------------+-------+
+| wohnfläche          | Size of dwelling in square meters                      | int   |
++---------------------+--------------------------------------------------------+-------+
+| gem_veranlagt       | Dummy: Married couple filing jointly for income tax    | bool  |
++---------------------+--------------------------------------------------------+-------+
+| in_ausbildung       | Dummy: Employment status "in education"                | bool  |
++---------------------+--------------------------------------------------------+-------+
+| alleinerziehend     | Dummy: Single parent                                   | bool  |
++---------------------+--------------------------------------------------------+-------+
+| bewohnt_eigentum    | Dummy: owner-occupied housing                          | bool  |
++---------------------+--------------------------------------------------------+-------+
+| immobilie_baujahr   | Construction year of dwelling                          | int   |
+|                     | (1: <1965,2:1966-2000,3:>2000)                         |       |
++---------------------+--------------------------------------------------------+-------+
+| prv_transfers_m     | Sum of monthly public/private transfers not simulated. | int   |
```

Comment:
Add a note that this will be split up soon.

Reply:
I made a small addition to the explanation in the documentation for this.